docs/animations/ParallaxService.md

```diff
@@ -43,14 +43,14 @@ MyUIElement.SetValue(ParallaxService.HorizontalMultiplierProperty, 0.5);
 
 [ParallaxService Sample Page Source](https://github.com/Microsoft/UWPCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ParallaxService). You can see this in action in [UWP Community Toolkit Sample App](https://www.microsoft.com/store/apps/9NBLGGH4TLCQ)
 
-## <a name="parallaxview"></a> Moving to ParallaxView
```

Comment:
There are external links that would be broken with this change, please revert back